apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -446,15 +448,35 @@ export const setCoteacherInvite = coteacherInvite => ({
   coteacherInvite,
 });
 
+export const setCoteacherInviteForPl = coteacherInviteForPl => ({
+  type: SET_COTEACHER_INVITE_FOR_PL,
+  coteacherInviteForPl,
+});
+
 export const asyncLoadCoteacherInvite = () => dispatch => {
   fetchJSON('/api/v1/section_instructors')
     .then(sectionInstructors => {
-      // Find the oldest invite.
-      const coteacherInvite = sectionInstructors.find(
-        instructor => instructor.status === 'invited'
-      );
+      const invitedPlSections = sectionInstructors.filter(section => {
```

Comment:
```suggestion
      const invitedPlSections = sectionInstructors.filter(instructorInvite => {
```
nit: this is a sectionInstructor object, not a section object.

Reply:
nit: if we are just trying to find the first invite that matches the filter, we can use `find` instead of `filter`

Reply:
Good eye!